[DOC] Fix format issue and refresh content (#5477)
opensearch-project/documentation-website: _dashboards/management/multi-data-sources.md

---
layout: default
title: Configuring and using multiple data sources
parent: Data sources
nav_order: 10
redirect_from: 
  - /dashboards/discover/multi-data-sources/
---

# Configuring and using multiple data sources

> The multiple data sources feature in OpenSearch Dashboards provides for the capability to ingest, process, and analyze data from a variety of sources. This feature is configured within the **Dashboards Management** > **Data sources** app. 

Reviewer: ```suggestion
You can ingest, process, and analyze data from multiple data sources in OpenSearch Dashboards. You configure the data sources in the **Dashboards Management** > **Data sources** app, as shown in the following image.
```

> A view of the main screen is shown in the following image.

Reviewer: ```suggestion

```

<img src="{{site.url}}{{site.baseurl}}/images/dashboards/data-sources-management.png" alt="Dashboards Management Data sources main screen" width="700">

## Getting started

The following tutorial guides you through configuring and using multiple data sources. 

### Step 1: Modify the YAML file settings

> To use the multiple data sources feature, the setting must be enabled. It is disabled by default. To enable multiple data sources:

Reviewer: ```suggestion
To use multiple data sources, you must enable the `data_source.enabled` setting. It is disabled by default. To enable multiple data sources:
```
